src/core/components/definitionList/definitionListItem/definitionListItem.tsx

```diff
@@ -14,15 +14,15 @@ export const DefinitionListItem: React.FC<IDefinitionListItemProps> = (props) =>
     return (
         <div
             className={classNames(
-                'flex w-full flex-col items-baseline justify-between gap-y-2 border-b border-neutral-100 py-3 last:border-none md:flex md:flex-row md:gap-x-6 md:py-4',
+                'flex flex-col gap-y-2 border-b border-neutral-100 py-3 last:border-none md:grid md:grid-cols-[1fr_2fr] md:gap-x-6 md:py-4',
                 className,
             )}
             {...otherProps}
         >
-            <dt className="line-clamp-1 shrink-0 text-base font-normal leading-relaxed text-neutral-800 md:line-clamp-6 md:w-40">
+            <dt className="line-clamp-1 text-base font-normal leading-relaxed text-neutral-800 md:line-clamp-none">
                 {term}
             </dt>
-            <dd className="size-full min-w-0 text-base font-normal leading-relaxed text-neutral-800">{children}</dd>
+            <dd className="text-base font-normal text-neutral-500">{children}</dd>
```

Comment:
This story under the `DefinitionList.Item` component looks broken, shouldn't we truncate the string in that case?
<img width="1088" alt="Screenshot 2024-11-18 at 10 30 26" src="https://github.com/user-attachments/assets/5984f817-131a-4691-99a9-82983b91215c">